### PR TITLE
Handle user with only System Administrator role

### LINKED ISSUE #1364: Odd behavior in-app if you're a system administrator with no permissions
### What is the expected behavior?
System Administrator can make changes to all parts of their Rise Vision account.
If a System Administrator cannot perform a task, an error is shown to explain it and direct them to a solution.

### What is the actual behavior?
A System Administrator has read-only access to all parts of the app.  In most cases there is either no error, or the error is unclear.  

### What steps can be taken to replicate the problem?
- Create a test account.
- Create a presentation, schedule, and Display
- Remove all permissions except for System Administrator from yourself
- Try to make changes to Presentations, Schedules, and Displays.  The behavior is inconsistent and the cause is not clear.

Here's a quick video overview:
https://drive.google.com/a/risevision.com/file/d/1v1RMmZ29Suaobx4G7Vng23iWgldOsVPe/view?usp=drivesdk

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Don't auto save Template if user is not Content Editor
Show status message in Template Editor to let user know they don't have permission
Disable display licensing toggle if user is not Display Administrator

## Motivation and Context
Fix #1364.

## How Has This Been Tested?
Manually tested locally and on stage-1.
Sample user: dipiwed551@xmail2.net
Sample Display: https://apps-stage-1.risevision.com/displays/details/4RDXFBWF5DU7
Sample Template: https://apps-stage-1.risevision.com/templates/edit/9838caf0-c634-4446-9761-5ccaa58f2034/

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
As it requires user with a special role setup, E2E test was skipped.

@alex-deaconu Please review. Thanks

### CODE REVIEW

File: web/scripts/template-editor/controllers/ctr-template-editor.js [hunk 1]
```diff
@@ -106,7 +113,7 @@ angular.module('risevision.template-editor.controllers')
       $scope.$on('presentationDeleted', _setUnsavedChanges.bind(null, false));
       $scope.$on('presentationPublished', _setUnsavedChangesAsync.bind(null, false));
 
-      $scope.$on('risevision.template-editor.brandingUnsavedChanges', autoSaveService.save);
+      $scope.$on('risevision.template-editor.brandingUnsavedChanges', _setUnsavedChangesAsync.bind(null, true));
```
Comment (alex-deaconu) [inline web/scripts/template-editor/controllers/ctr-template-editor.js]: @ezequielc curious, is this related to the fix? Why is this change necessary? Thanks.

File: web/scss/sections/template-editor-layout.scss [hunk 2]
```diff
@@ -122,7 +122,7 @@ $maderio-green-disabled: #a2dbb2;
 
     .auto-saving-changes {
       position: relative;
-      width: 120px;
+      width: 180px;
```
Comment (alex-deaconu) [inline web/scss/sections/template-editor-layout.scss]: @ezequielc do you think this would cause issues with the layout at different resolutions? Thanks.

File: web/scripts/template-editor/controllers/ctr-template-editor.js (hunk 1 above)
Comment (ezequielc) [reply web/scripts/template-editor/controllers/ctr-template-editor.js]: @alex-deaconu Yes, it is related to the fix. It reuses `_setUnsavedChanges` where it checks for the required role before saving: https://github.com/Rise-Vision/rise-vision-apps/pull/1374/files#diff-783c85de82bac6b97f45ca0ca36c88b6R64 . It avoids code duplication. Thanks

File: web/scss/sections/template-editor-layout.scss (hunk 2 above)
Comment (ezequielc) [reply web/scss/sections/template-editor-layout.scss]: @alex-deaconu I checked multiple resolutions, resizing the window and with Publish button being present or not and they all looked good. No concerns from me here. Thanks

Comment (alex-deaconu) [reply web/scss/sections/template-editor-layout.scss]: @ezequielc I've double checked and I don't see any issues with the updated with. Thanks!
